src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -419,7 +418,7 @@ bool EmitAnchors()
             }
 
             // Emits a case-sensitive left-to-right prefix search for a string at the beginning of the pattern.
-            void EmitIndexOf_LeftToRight(string prefix)
+            void EmitIndexOf(string prefix)
```

Comment:
NIT: should we update the comment to to remove the `left-to-right` part even though it is technically true? I guess keeping it might imply that there is also a right-to-left somewhere but I'm fine if you want to keep it

Reply:
fixed